gatsby-node.js

```diff
@@ -1,9 +1,71 @@
-/**
- * Implement Gatsby's Node APIs in this file.
- *
- * See: https://www.gatsbyjs.org/docs/node-apis/
- */
-const sharp = require("sharp")
+const chalk = require('chalk');
 
+const sharp = require("sharp")
 sharp.cache(false)
-sharp.simd(false)
\ No newline at end of file
+sharp.simd(false)
+
+exports.createPages = ({ graphql, actions }) => {
+    const { createRedirect } = actions
```

Comment:
Added by that zeit plugin?

Reply:
No, that's for generating client-side redirects. 'gatsby-plugin-zeit-now' is used for the server-side